notebooks/data analysis/Pathfinder_system_abilities_attributes.ipynb

```diff
@@ -0,0 +1,1915 @@
+    "### Analiza attributes: *resistances*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 235,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "0                                                    NaN\n1                                                    NaN\n2                                                    NaN\n3                                                    NaN\n4                         [{'type': 'fire', 'value': 3}]\n                             ...                        \n360                    [{'type': 'mental', 'value': 20}]\n361                                                  NaN\n362    [{'exceptions': ['adamantine', 'bludgeoning'],...\n363    [{'doubleVs': ['non-magical'], 'exceptions': [...\n364                                                  NaN\nName: resistances, Length: 1206, dtype: object"
+     },
+     "execution_count": 235,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attributes.resistances"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 236,
+   "outputs": [],
+   "source": [
+    "not_null_resistances = attributes[pd.notnull(attributes.resistances)].resistances"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 237,
+   "outputs": [],
+   "source": [
+    "unique_resistances = unique_val_list(not_null_resistances, \"type\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 238,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "defaultdict(<function __main__.unique_val_list.<locals>.<lambda>()>,\n            {'fire': 66,\n             'all-damage': 30,\n             'bludgeoning': 47,\n             'piercing': 70,\n             'cold': 45,\n             'poison': 39,\n             'electricity': 34,\n             'slashing': 63,\n             'acid': 21,\n             'physical': 81,\n             'precision': 7,\n             'protean-anatomy': 6,\n             'mental': 35,\n             'negative': 12,\n             'sonic': 8,\n             'force': 1,\n             'critical-hits': 2,\n             'evil': 1,\n             'air': 1,\n             'earth': 1,\n             'plant': 1,\n             'water': 1})"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "all resistances 572\n",
+      "unique values: 22\n"
+     ]
+    }
+   ],
+   "source": [
+    "display(unique_resistances)\n",
+    "print(\"all resistances\", sum(unique_resistances.values()))\n",
+    "print(\"unique values:\", len(unique_resistances))"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "*To też można uwzględnić*\n",
```

Comment:
Dodałabym do kategorii na potem.

W dnd jest to trochę łatwiejsze do uwzględnienia... 
Bo weakness i resistance to są przeciwieństwa i ja myślałam że je będziemy oznaczać jako -val +val. Ale całkiem sporo rzeczy się nie powtarza. I myślałam że tu będą tylko typy obrażeń, ale też nie.